hardhat.config.js

```diff
@@ -21,6 +21,10 @@ module.exports = {
       accounts: {
         count: 1,
       },
+      mining: {
```

Comment:
Include a comment explaining why you are doing this.

```suggestion
      // Disable auto-mining so that e2e tests can explicitly test pending states.
      mining: {
```